packages/triggers-calculations/src/get-morphoblue-position.ts

```diff
@@ -65,34 +81,43 @@ export async function getMorphoBluePosition(
         functionName: 'symbol',
       },
     ],
-    allowFailure: true,
+    allowFailure: false,
+  })
+  const [oraclePriceTemp] = await publicClient.multicall({
+    contracts: [
+      {
+        abi: chainlinkPairOracleAbi,
+        address: oracleAddress,
+        functionName: 'price',
+      },
+    ],
+    allowFailure: false,
   })
 
-  const collateralAmount = collateralData.status === 'success' ? collateralData.result[0] : 0n
-  const debtAmount = debtData.status === 'success' ? debtData.result[1] + debtData.result[2] : 0n
+  const collateralPriceInDebt =
+    (oraclePriceTemp * 10n ** 18n) /
+    10n ** (36n + BigInt(debtDecimals) - BigInt(collateralDecimals))
```

Comment:
`Returns the price of 1 asset of collateral token quoted in 1 asset of loan token, scaled by 1e36.`
We have the `PRICE_DECIMALS` defined somewhere, and the function that calculates the LTV needs the price in the correct precision.
We have a function to change the decimals: 
https://github.com/OasisDEX/summerfi-monorepo/blob/dev/packages/triggers-calculations/src/helpers/normalize-balance.ts#L7

Usage in that case: 
```
const collateralPriceInDebt = normalizeAmount(oraclePriceDebt, 36 + debtDecimals - collateralDecimals, PRICE_DECIMALS)

```
For the values (36) I used an oracle https://etherscan.io/address/0x2a01EB9496094dA03c4E364Def50f5aD1280AD72#readContract for wstETH/ETH market. We need to check if other oracles have maybe different precisions.